buildSrc/DevBuild.js

```diff
@@ -181,6 +181,10 @@ globalThis.buildOptions.sqliteNativePath = "./better-sqlite3.node";`,
 	await runStep("Desktop: assets", async () => {
 		const desktopIconsPath = "./resources/desktop-icons"
 		await fs.copy(desktopIconsPath, "./build/desktop/resources/icons", { overwrite: true })
+		await fs.move("./build/desktop/resources/icons/logo-solo-dev.png", "./build/desktop/resources/icons/logo-solo-red.png", { overwrite: true })
```

Comment:
okay sorry I just realized we overwrite an icon instead of just specifying another one few lines below in `iconPath`

Reply:
I think so, why should we overwrite red logo with non-red one and then reference the red one when we could just reference the right one?

Reply:
We actually can't do that because there is no way to know if it is dev build in `DesktopIntegrator`, unless I missed something?